src/styles/components/committees/committeeDescription.module.scss

```diff
@@ -0,0 +1,57 @@
+.wrapper{
+    height: auto;
+    background: var(--off-white, #FFFBF0);
+    padding-top: 4rem;
```

Comment:
you don't need the padding on the top, it adds a lot of empty space on the page which is not in the design. also paddings and margins should always be in px.